### ISSUE TITLE
Short descriptions for menu items

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/docker-drupal#84]
We shoudl add tooltip explanations for the menu items (e.g. what is a showcase)

**Comment by claudiahahn** [comment]
Suggestions: 
**solution offers**: 
Products or services to support climate resilient planning (e.g. software tools, consulting services, or novel adaptation options) 

**showcases:** 
Case studies demonstrating the application of products or services in a specific location / examples of climate resilient urban planning 

If showcases should always relate to a solution offer, than the last part (examples of climate resilient urban planning) needs to be deleted. However, I think it would be beneficial for users to also see best / good practice examples?

**Comment by p-a-s-c-a-l** [comment]
Here is the "[official definition](https://github.com/clarity-h2020/csis-architecture/blob/dev/src/main/markdown/CSIS%20and%20the%20Marketplace%20concepts.md)":

### A Solution Offer

A Solution Offer is a (generic) product or service that can be advertised in the Marketplace and the CSIS and thus gives Solution Providers the possibility to promote their products and services. A Solution Offer can apply to one or more (Economic) Sectors and EU-GL step/modules.

Solution Offers can consist of different Climate Services (advice, software) or represent a distinct Climate Service on their own. The screening tool part of the CSIS is an example of a Solution Offer. Another example would be a service for creating tailored Data Packages.

Additionally, Solution Offers also represent physical products and services like novel reflective materials, CO2-absorbant paint, climate-change-aware construction services, etc. Interestingly, such a physical Solution Offer could be used as Adaptation Option.

### Showcase

A Showcase is a concrete application of one or more Solution Offers, e.g. a concrete application of an Expert Service in a specific location or an implementation / usage of a tailored or generic Tool or Product (Climate Service) in a specific location.

The CSIS Screening Study "Adaptation Scenarios for Metropolitan Resilience Planning in the Naples metropolitan area" is a Showcase of the CSIS Screening Tool used in conjunction with a tailored Data Package (Heat Wave and Pluvial Flood Hazards, Population and Urban Infrastructure Elements at Risk, . etc.). The usage of a specific physical product, e.g. painting a 1.000m² wall building with C02-absorbant paint in the city of Rome is another Showcase (of the Solution Offer "C02-absorbant paint product"). More specifically, this particular Showcases demonstrates how the product is actually used as Adaptation Option.

![grafik](https://user-images.githubusercontent.com/1788226/62136064-a958cc00-b2e3-11e9-9bdd-152bb460c6a8.png)

**Comment by claudiahahn** [comment]
According to the "official definition" a showcase should  always be related to a solution offer. 

In that case, we can only enter showcases ourself, if there is a generic tool (free) that is being used and which we have already listed under solution offers. 
**The description could be: "Application of solution offers (products or services to support climate resilience) in a specific location"**

I guess it is on us whether we want to include examples of climate resilient planning or not. If we want to do that and still want to have the relationship to the solution offers, we could also list Oppla or the Climate Adapt Adaptation Support Tool as solution offers, and take examples from there. Then we should mention: "more such examples can be found here: xxx". 
However, if there are other interesting examples, like https://inhabitat.com/copenhagens-enghaveparken-public-park-is-designed-to-be-flooded/, or https://www.publicspace.org/works/-/project/h034-water-square-in-benthemplein, that are not listed under e.g. Oppla, we could not include them. 

So if we want to leave it open (and change the "official definition"):
**Alternative description: "Application of solution offers (products or services to support climate resilience) in a specific location & examples of climate resilient planning"**

It depends on whether we want to (additionally) also select and present good practice examples or not.

**Comment by claudiahahn** [comment]
The best way would probably be to encourage consulting companies that were involved in, for example, the above mentioned projects to enter such showcases and their solution offers (services) themself.

**Comment by p-a-s-c-a-l** [comment]
> we could also list Oppla or the Climate Adapt Adaptation Support Tool as solution offers

No Oppla and Climate Adapt Adaptation Support Tool  are no Solution Offers. The climate resilient planning examples (Showcases) are the result of the application of some expert (climate) service, so the expert services are the solution offer. It's the idea of marketplace that CS providers can describe their (generic) CS as solution offers in the marketplace and CSIS, respectively. Solution Offers are both used in Marketplace and CSIS.

Example: https://inhabitat.com/copenhagens-enghaveparken-public-park-is-designed-to-be-flooded/ is a **Showcase**. What's the related Solution Offer?

"*A consulting group consisting of COWI, TREDJE NATUR and Platant proposed the project*" ...

=> the solution offer(s) are the consulting services related to "designing resilient architecture" provided by those companies. So the companies have to register as Solution Providers in our [Marketplace](https://github.com/clarity-h2020/marketplace/issues), describe their service offerings in terms of Solution Offers and then we can add showcases like the one mentioned above to advertise their CS in CSIS.

**Comment by claudiahahn** [comment]
That's what I meant, with respect to the example. 

From the "official definition" I also thought solution offers could be free of charge as well. 
That's why I thought the Climate Adapt Urban Adaptation Support Tool could potentially also be listed, since its aim is to assist cities, towns and local authorities. But its fine for me if we try to avoid listing too many "tools" that are mainly guidelines.

**Comment by claudiahahn** [comment]
from my side - final description:
**solution offers**:
Products or services to support climate resilient planning (e.g. software tools, consulting services, or novel adaptation options)

**showcases**:
Application of solution offers (products or services to support climate resilience) in a specific location

**Comment by p-a-s-c-a-l** [comment]
@DanielRodera did you add the tooltips?

**Comment by DanielRodera** [comment]
Hi @p-a-s-c-a-l,

I have added the tooltips for the menu items (short description) with the text proposed by @claudiahahn,

I also added a paragraph (with the official definition described above) for the Solution offers and showcases at the beginning of the view. In my opinion, these paragraphs help the users to clarify the content they are seeing. If you think is not necessary let me know and I will remove them.

Thanks
